test_docker.sh

```diff
@@ -1 +0,0 @@
-docker run --rm -v $(pwd):/Qt.py mottosso/qt.py27
```

Comment:
Aaaand finally this one. I just them all the time. Also it simplifies the docs, see the last section in the README on testing.